webpack/InsightsCloudSync/InsightsCloudSyncReducers.js

```diff
@@ -0,0 +1,3 @@
+const reducers = {};
```

Comment:
since we are not updating the redux store with some data,
I would be OK with avoiding the use of redux on this early stage.. unless we are planning to add more components to the page

Reply:
I suppose we will need to add more components later. I think leaving this boiler plate code makes sense.